src/black/parsing.py

```diff
@@ -175,11 +180,17 @@ def parse_ast(src: str) -> Union[ast.AST, ast3.AST]:
     first_error = ""
     for version in sorted(versions, reverse=True):
         try:
-            return parse_single_version(src, version)
+            return parse_single_version(src, version, type_comments=True)
         except SyntaxError as e:
             if not first_error:
                 first_error = str(e)
 
+            # Try to parse without type comments
```

Comment:
Need to think more about whether we should have a separate loop instead. I *think* that should never affect behavior, just performance.

Reply:
I'll move it to an outer for loop anyway. Just for guaranteed backwards compatibility.